builder/build_steps/build_app.yaml.in

```diff
@@ -1,6 +1,6 @@
 steps:
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '--pull', '--no-cache', '-t', '$IMAGE', '-f', 'build_app/Dockerfile', '.']
+    args: ['build', '--pull', '--no-cache', '-t', 'gcr.io/$PROJECT/ruby-build-app:$TAG', '-f', 'build_app/Dockerfile', '.']
```

Comment:
The [GCR guide](https://cloud.google.com/container-registry/docs/pushing) states `us.gcr.io` is preferred over `gcr.io`. I think we should follow the instruction and make our test images using `us.gcr.io`registry.